05-UI-ReactJS/source/js/components/views/movie-form.js

```diff
@@ -0,0 +1,88 @@
+import React, { PropTypes } from "react";
+import Movie from "../../movie.js";
+import store from "../../store";
+import { browserHistory } from 'react-router';
```

Comment:
I'm using it [here](https://github.com/julian-aiup/startup/blob/topic5/05-UI-ReactJS/source/js/components/views/movie-form.js#L80), because I have a button that redirects to other page